com.unity.render-pipelines.core/Runtime/Debugging/DebugManager.cs

```diff
@@ -241,20 +249,34 @@ void CheckPersistentCanvas()
             }
         }
 
-        internal void TogglePersistent(DebugUI.Widget widget)
+        internal void TogglePersistent(DebugUI.Widget widget, int? forceTupleIndex = null)
         {
             if (widget == null)
                 return;
 
-            var valueWidget = widget as DebugUI.Value;
-            if (valueWidget == null)
+            CheckPersistentCanvas();
+
+            switch (widget)
             {
-                Debug.Log("Only DebugUI.Value items can be made persistent.");
-                return;
+                case DebugUI.Value value:
+                    m_RootUIPersistentCanvas.Toggle(value);
+                    break;
+                case DebugUI.ValueTuple valueTuple:
+                    m_RootUIPersistentCanvas.Toggle(valueTuple, forceTupleIndex);
+                    break;
+                case DebugUI.Container container:
+                    // When container is toggled, we make sure that if there are ValueTuples, they all get the same element index.
+                    int pinnedIndex = container.children.Max(w => (w as DebugUI.ValueTuple)?.pinnedElementIndex ?? -1);
+                    foreach (var child in container.children)
+                    {
+                        if (child is DebugUI.Value || child is DebugUI.ValueTuple)
+                            TogglePersistent(child, pinnedIndex);
+                    }
+                    break;
+                default:
+                    Debug.Log("Only readonly items can be made persistent.");
```

Comment:
I wonder if this should be better a `warning`

Reply:
I'll leave it as Log as it's reaction to user input, it's just info that it's not supported.